pasqal-cloud/pasqal_cloud/project.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+
+
+class Project(BaseModel):
```

Comment:
Our other pydantic models allow extra fields, I think this can be useful to add here as well: 

```    
model_config = ConfigDict(extra="allow")
 ```